src/Aspire.Cli/Commands/McpStartCommand.cs

```diff
@@ -108,7 +109,7 @@ private async ValueTask<CallToolResult> HandleCallToolAsync(RequestContext<CallT
         if (_tools.TryGetValue(toolName, out var tool))
         {
             // Handle tools that don't need an MCP connection to the AppHost
-            if (toolName is "select_apphost" or "list_apphosts" or "list_integrations" or "get_integration_docs")
+            if (toolName is "select_apphost" or "list_apphosts" or "list_integrations" or "get_integration_docs" or "doctor")
```

Comment:
This is existing code, but it is gross. The tool should tell it if it needs to be connected to the AppHost.